File: answer.py
def number(bus_stops):
    return sum([stop[0] - stop[1] for stop in bus_stops])
  
def number(stops):
    return sum(i - o for i, o in stops)
  
def number(bus_stops):
    total_people = 0
    for people in bus_stops:
        total_people = total_people+people[0]-people[1]
    return total_people

File: answer.js
var number = function(busStops){
  return busStops.map(x=>x.reduce((peopleIn,peopleOut)=>peopleIn-peopleOut)).reduce((accumulator,currentValue)=>accumulator+currentValue)
}

const number = (busStops) => busStops.reduce((rem, [on, off]) => rem + on - off, 0)

var number = function(busStops){
	var totalPeople = 0;
  for (var i=0; i<busStops.length; i++) {
  	totalPeople += busStops[i][0];
    totalPeople -= busStops[i][1];
  }
  return totalPeople;
}

const number = busStops => busStops.reduce((p,n) => p+n[0]-n[1],0)
